debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/FieldRenamesTest.java

```diff
@@ -0,0 +1,1113 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb;
+
+import io.debezium.schema.TopicSelector;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.bson.BsonTimestamp;
+import org.bson.Document;
+import org.bson.json.JsonMode;
+import org.bson.json.JsonWriterSettings;
+import org.bson.types.ObjectId;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static io.debezium.data.Envelope.FieldName.AFTER;
+import static org.fest.assertions.Assertions.assertThat;
+
+public class FieldRenamesTest {
```

Comment:
Two questions on the renaming behavior:

* What happens if two renames have the same target field? I think the code raises an exception, is there a test for it, though?
* Can the source of a second rename expression apply to the target name of a first rename? Or do they all apply to the original field names (which is what I'd expect)?

Reply:
> What happens if two renames have the same target field? I think the code raises an exception, is there a test for it, though?

The code raises an exception, I added tests for this case.

> Can the source of a second rename expression apply to the target name of a first rename?

Yes, it can. But for simplicity, you can specify the deepest rename expression first, so that you don't need to consider the previous renaming.

Reply:
> Yes, it can. But for simplicity, you can specify the deepest rename expression first, so that you don't need to consider the previous renaming.

Ok, this should be clarified in the docs then. In fact I was expecting the other behavior, but that's fine.  On a more general note, can you provide a PR for adding the new options to the MongoDB connector docs?